modular_skyrat/code/modules/research/xenoarch/xenobotany/grown/vaporsac.dm

```diff
@@ -0,0 +1,28 @@
+/obj/item/seeds/vaporsac
```

Comment:
Do all these plants really need their own .dm files? I suggest keeping it all in a single .dm for simplicity and ease of use, something like xenoarchflora.dm

Reply:
The plants are actually done using this, see the hydroponics module.

Reply:
Ah, wasn't aware of that, my bad.